implementation.js

```diff
@@ -19,11 +19,21 @@ var slice = Function.call.bind(Array.prototype.slice);
 var concat = Function.call.bind(Array.prototype.concat);
 var forEach = Function.call.bind(Array.prototype.forEach);
 
-var hasSymbols = require('has-symbols')();
+var hasSymbols = typeof global.Symbol !== 'function';
 
 var kCustomPromisifiedSymbol = hasSymbols ? Symbol('util.promisify.custom') : null;
 var kCustomPromisifyArgsSymbol = hasSymbols ? Symbol('customPromisifyArgs') : null;
 
+var getOwnPropertyDescriptors = function _getOwnPropertyDescriptors(obj) {
+	var out = {};
+	var keys = Object.keys(obj);
```

Comment:
this means the implementation is reliant on nobody doing `delete Object.keys`

Reply:
:(